packages/webapp/src/routes/land/clients-name.js

```diff
@@ -0,0 +1,39 @@
+import constants from '../../utils/constants.js'
+import {
+  validateFirstLastName
+} from '../../utils/helpers.js'
+import isEmpty from 'lodash/isEmpty.js'
+
+const handlers = {
+  get: async (request, h) => {
+    return h.view(constants.views.CLIENTS_NAME)

```

Comment:
Can the GET populate the name fields if pre exisiting, just to help for routing from CYA page?

Reply:
done